QUEST_LV_0100.tsv

```diff
@@ -8095,7 +8095,7 @@ QUEST_LV_0100_20150908_008094
 QUEST_LV_0100_20150908_008095	
 QUEST_LV_0100_20150908_008096	
 QUEST_LV_0100_20150918_008097	This place became dangerous because of monsters emerging since Medis Diena.{nl}Quickly return while you can, because Kepa's will soon be marching in.
-QUEST_LV_0100_20150918_008098	You were telling that we need to assemble, right?{nl}Please search my luggage first and return quickly, so I'll go as soon as I can.
+QUEST_LV_0100_20150918_008098	You were telling that we need to assemble, right?{nl}Please search my bag first and return quickly, so I'll go as soon as I can.
```

Comment:
Soukyuu has this covered already. I believe he made a more fitting term for it, "belongings". 
Check his PR.